components/media-player/MediaPlayerOpen.vue

```diff
@@ -24,7 +26,23 @@ const {
   rewind,
   fastForward,
   repeatStatus,
+  moveTrack
 } = useNuxtApp().$mediaPlayer;
+
+function isCurrentTrack(track: TrackModel) {
```

Comment:
This means that if I have a track in the queue twice, both will be marked as playing, or paused... Can we fix this?